### ISSUE TITLE
sudo pip command not found

### ISSUE THREAD

**Issue by jamesbrunke** [Marten4n6/EvilOSX#88]
Im probably doing something stupid, but when I run this command it comes back that it can't be found. Should I already have python installed for this to work?

**Comment by kodifiera99** [comment]
Yes. This is a Python project, so how could it run without Python? This is a nonissue

**Comment by ChloeLeitch** [comment]
You have to install pip, if your version of python does not come with pip.
How to install on Mac: https://www.youtube.com/watch?v=yBdZZGPpYxg
How to install on Windows: https://www.makeuseof.com/tag/install-pip-for-python/
